singularity-eos/eos/eos_stellar_collapse.hpp

```diff
@@ -291,7 +294,7 @@ class StellarCollapse : public EosBase<StellarCollapse> {
   static constexpr Real TNormal_ = 5 * GK2MeV_; // Threshold of NSE
   static constexpr Real rhoNormal_ = 2.e12;     // 1./100'th of nuclear density
   static constexpr Real YeNormal_ = 0.3517;     // Beta equilibrium value
-  Real sieNormal_, PNormal_;
+  Real sieNormal_, PNormal_, SNormal_;
```

Comment:
I think the `SNormal_` stuff is not needed... But it's nice you added it. May come in handy later.